ambry-clustermap/src/main/java/com/github/ambry/clustermap/RecoveryTestClusterManager.java

```diff
@@ -59,6 +59,7 @@ public PartitionId getPartitionIdFromStream(InputStream stream) throws IOExcepti
     } catch (IOException e) {
       // for composite blob, it has the list of Blob IDs. It may not exist in the static cluster map.
       // swallow the exception and try with helixClusterManager
+      logger.error("staticClusterManager getPartitionIdFromStream throws {}", e.getMessage());
```

Comment:
You mentioned this would overwhelm the log ? The log.error is good but I suggested putting this inside an if-case to prevent overwhelming the log like this 
```
if (e.getMessage.equals("...") == true or false){ 
    log.error("...")
}
```

Reply:
I think this is the only exception message we have so far. So I guess we'll log the same amount of error messages.

Reply:
I see, so this won't overwhelm the log ?

Reply:
ok understood